_includes/footer.html

```diff
@@ -1,30 +1,67 @@
 <footer class="usa-footer site-footer" role="contentinfo">
-  <div class="footer-section-bottom bg-base-lighter">
-    <div class="grid-container">
-      <div class="grid-row padding-3">
-        <div class="tablet:grid-col">
-          <div class="logo-links">
-            <a class="footer-logo media_link" href="https://cloud.gov">
-              {% image_with_class "./_img/logo-img.png" "width-10 height-auto" "Agency logo" %}
-            </a>
-          </div>
+  <hr class="footer-border">
+  <div class="usa-identifier site-identifier">
+    <section class="usa-identifier__section usa-identifier__section--masthead" aria-label="Agency identifier">
+        <div class="usa-identifier__container">
+            <div class="usa-identifier__logos">
+                <a href="javascript:void(0);" class="usa-identifier__logo">
+                    {% image_with_class footer.image.src footer.image.class footer.image.alt %}
```

Comment:
because these image values are tied to this include only, I'd pass them inline (e.g. `{% image_with_class "_img/gsa-logo.svg", ...`). Then there's no need to maintain a separate data file.

Also both the link and the image have the `usa-identifier__logo` class (I think only the image needs)